neurodsp/sim/combined.py

```diff
@@ -82,3 +84,89 @@ def sim_combined(n_seconds, fs, components, component_variances=1):
     sig = np.sum(sig_components, axis=0)
 
     return sig
+
+
+@normalize
+def sim_central_freq(n_seconds, fs, central_freq, bw, ht, ap_func='sim_powerlaw',
```

Comment:
I _think_ maybe we can change the name of this function? I don't find `sim_central_freq` to be particularly obvious (in terms of what this does). Annoyingly, there's not an obvious alternative. 

I think I would be tempted, with a bit of a call-out to FOOOF, to call it something like `sim_peak_oscillation`, or perhaps similar, with the idea that it "simulates an oscillation that exhibits as a peak in the power spectrum, with a designated center frequency, power / height, and bandwidth. 

Thoughts? Any other name suggestions?

Reply:
`sim_peak_oscillation` works for me. `sim_gaussian` could also work.